query-node/mappings/src/content/app.ts

```diff
@@ -33,8 +33,7 @@ export async function processCreateAppMessage(
   const newApp = new App({
     name,
     id: appId,
-    ownerMember: memberId ? new Membership({ id: memberId }) : undefined,
```

Comment:
I can't comment on line 16 on GitHub, but there is a redundant condition and a comment. 
```tsx

  // memberId is required for MemberRemark, but not for LeadRemark
  if (event.method === 'MemberRemarked' && !memberId) {
    inconsistentState("memberId wasn't provided")
  }
```
We should update it as well